resources/gm9/scripts/GM9Megascript.gm9

```diff
@@ -1175,6 +1175,9 @@ end
 if	not allow -a 0:/
  	goto MainMenu_Scripts_from_Plailect's_Guide
 end
+if      not allow -a A:/
```

Comment:
Indent?

Reply:
Ugh, indent. That was a tab/space mixing oversight on my part. Thanks for pointing it out, but I don't think I'll bother with a whole new PR just to fix that because it's only a minor eye sore and the scripting language doesn't care about indents at all.

Reply:
@aspargas2 - maybe fix it locally and include the indent fix in the next script related pr you do?

Reply:
Yeah, I'll do that.